terraform/webdav.tf

```diff
@@ -0,0 +1,40 @@
+resource "heroku_app" "webdav" {
+  name   = "dandidav"
+  region = "us"
+  acm    = true
+
+  organization {
+    name = data.heroku_team.dandi.name
+  }
+
+  buildpacks = [
+    "https://buildpack-registry.s3.amazonaws.com/buildpacks/emk/rust.tgz"
+  ]
+}
+
+resource "heroku_formation" "webdav_heroku_web" {
+  app_id   = heroku_app.webdav.id
+  type     = "web"
+  size     = "basic"
+  quantity = 1
+}
+
+# Enable the runtime-dyno-metadata feature for the Django app
+# https://devcenter.heroku.com/articles/dyno-metadata
```

Comment:
I'm guessing this is to make the sha available to the binary? IMO this is a what-not-why comment.

Reply:
Yes. I adjusted that comment to explain the "why" instead.